{"event": "runner_on_failed", "uuid": "dc4d3b10-5ca6-4dc5-b680-e0a4b69b6ef1", "stdout": "\u001b[0;31mfatal: [ubuntu@54.193.67.251]: FAILED! => {\"msg\": \"the connection plugin 'ubuntu@54.193.67.251' was not found\"}\u001b[0m", "counter": 6, "pid": 30684, "created": "2019-09-24T14:56:40.459287", "end_line": 6, "runner_ident": "b02652f3-aa42-4870-a8cc-3477da134c42", "start_line": 5, "event_data": {"play_pattern": "ubuntu@54.193.67.251", "play": "ubuntu@54.193.67.251", "event_loop": null, "task_args": "", "remote_addr": "ubuntu@54.193.67.251", "res": {"msg": "the connection plugin 'ubuntu@54.193.67.251' was not found", "_ansible_no_log": false}, "pid": 30684, "play_uuid": "080027d7-8737-616d-60c0-000000000006", "task_uuid": "080027d7-8737-616d-60c0-00000000000e", "task": "Gathering Facts", "playbook_uuid": "a91c1cbf-8fc0-4be9-a9e6-f90375140282", "playbook": "test.yml", "task_action": "gather_facts", "host": "ubuntu@54.193.67.251", "ignore_errors": null, "task_path": "/home/tardis/Programming/Ansible/ansible-runner/demo/project/test.yml:1"}, "parent_uuid": "080027d7-8737-616d-60c0-00000000000e"}